chemicaltoolbox/autodock_vina/prepare_box/prepare_box.xml

```diff
@@ -0,0 +1,37 @@
+<tool id="prepare_box" name="Calculate the box parameters for an AutoDock Vina job from an input mol file (confounding box)" version="0.1.0">
+    <requirements>
+        <requirement type="package">rdkit</requirement>
+        <requirement type="package">numpy</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python $__tool_directory__/calc_vina_box_params.py --ligand '$input1' --config '$output1' --bufx 1 --bufy 2 --bufz 3
+    ]]></command>
+    <inputs>
+        <param type="data" name="input1" format="mol" />
+    </inputs>
+    <outputs>
+        <data name="output1" format="txt" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input1" value="NUDT5A-x0114_2.mol"/>
+            <output name="output1" file="box_params.txt"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+        usage: calc_vina_box_params.py [-h] [--ligand LIGAND_PATH] [--config OUTPUT]
+                               [--tol TOLERANCE] [--exh EXHAUSTIVENESS]
+                               [--bufx BUFX] [--bufy BUFY] [--bufz BUFZ]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --ligand LIGAND_PATH
+  --config OUTPUT
+  --tol TOLERANCE
+  --exh EXHAUSTIVENESS
+  --bufx BUFX
+  --bufy BUFY
+  --bufz BUFZ
+
+    ]]></help>
```

Comment:
maybe adding a citation for rdkit?

Reply:
resolved by ce67e4feeb7c8b593ce563f893b714d6c4cbdfcc